package.json

```diff
@@ -4,6 +4,11 @@
   "description": "Cumulus Framework for ingesting and processing NASA Earth data streams",
   "scripts": {
     "audit": "audit-ci --config ./audit-ci.json && lerna exec -- audit-ci --config $(pwd)/audit-ci.json",
+    "deploy": "npm run deploy-cumulus",
```

Comment:
My leaning is to not include these additional scripts. If we do, though, then it seems like they belong in `example` rather than at the top level.

Reply:
I considered putting them in `example`, but that would then require the "precumulus-deploy" to cd to the parent directory: `"cd .. && npm install && npm run bootstrap"`. This isn't terrible, but I felt that it was mildly more odd than this putting it at the top level.

However, if the consensus is to put them in `example`, that's fine.

Of course, if there is a stronger preference to not include them at all, then my question would be why not? These don't appear to be any less appropriate than some of the existing ones, unless, of course, everybody prefers slightly different steps, in which case they aren't generally usable.